zomes/hc_zome_trust_atom/src/trust_atom.rs

```diff
@@ -32,7 +32,8 @@ const LINK_TAG_ARROW_REVERSE: [u8; 3] = [226, 134, 169]; // Unicode "↩" // hex
 #[hdk_entry(id = "extra", visibility = "public")]
 #[derive(Clone)]
 pub struct Extra {
-  pub fields: BTreeMap<String, String>, // extra content
+  pub content_overflow: Option<String>,
+  pub extra_fields: Option<BTreeMap<String, String>>, // extra content
```

Comment:
would just be empty if nothing in there:
```suggestion
  pub extra_fields: BTreeMap<String, String>, // extra content 
```